wiz-outpost-lite/values.yaml

```diff
@@ -74,16 +74,52 @@ terminationGracePeriodSeconds: 30
 serviceAccount:
     create: false
 
+podSecurityContext: {}
+containerSecurityContext:
+  capabilities:
+    add:
+    - SYS_ADMIN
+  seLinuxOptions:
+    type: spc_t
+
+# Runner type definitions - these define defaults for each type of runner
```

Comment:
The name and flattening collides and is misaligned w/ the runners.\<x> hierarchy below.

We already offer per-runner overrides, we now need module overrides, where the value precedence is (runner, module, global)